### ISSUE TITLE
ERR! enable Couldn’t find a repo with this slug.

### ISSUE THREAD

**Issue by baptistebriel** [greenkeeperio/greenkeeper#41]
Hello,
I have this error when typing `greenkeeper enable`:

```
🌴  info enable The GitHub slug is: BaptisteBriel/biggie
🌴  ERR! enable Couldn’t find a repo with this slug.
🌴  ERR! enable Either it doesn’t exist on GitHub, it’s a fork, or private.
```

But the repo exists, within the branch `gh-pages` - do you have any idea? I am in the right folder.
Thanks!

**Comment by baptistebriel** [comment]
Ok, related to #4! Sorry :)

**Comment by rstuven** [comment]
@baptistebriel I'm having the same issue. Would you tell me how did you solve it, please?

**Comment by baptistebriel** [comment]
Closed because duplicate of #37 - still waiting for more detailed explanations. :)
